### DISCUSSION TITLE
Carousel mouse drag on desktop

### DISCUSSION THREAD

**Discussion by AYGTX** [primefaces/community#776] [PrimeNG] [answered]
I observed that the p-carousel component enables swiping through slides on mobile devices, but this functionality is not available on desktop. It would be great if the component could also support slide navigation through mouse dragging on desktop.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback, we've added it to our internal backlog.

**Comment by BrachiWeiss** [comment]
Hi,
I use your library, and I lack the option of dragging.
Is this supposed to be added soon?
and RTL support
thanks
